builtin-functions/kphp-light/functions.txt

```diff
@@ -160,6 +160,53 @@ function component_server_fetch_request($query ::: ComponentQuery) ::: string;
 /** @kphp-extern-func-info interruptible */
 function component_server_send_response($query ::: ComponentQuery, $message ::: string) ::: void;
 
+// === Job Worker =================================================================================
```

Comment:
Let's create file `job-worker.txt` with interface and include it in functions.txt